testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/util/UIUtils.java

```diff
@@ -69,11 +68,35 @@ public static void refreshPageAndWaitForLoad() {
         performOperationWithPageReload(() -> getCurrentDriver().navigate().refresh());
     }
 
+    /**
+     * The method executes click or sendKeys(Keys.ENTER) in the element.
```

Comment:
Is there a bug report for this behaviour in Chrome? If so, can we link it here for future reference?

Reply:
I didn't find a bug, although I didn't search a lot. Maybe is something between chrome, selenium, arquillian,... We can test later and just changing back the clickLink method would be enough.